cli/messages.go

```diff
@@ -160,6 +160,19 @@ Example: nordvpn set meshnet on`
 	MsgMeshnetAllowLocalNetworkUsage          = "Allow the peer to access local network when routing traffic through this device."
 	MsgMeshnetAllowFileshare                  = "Allow the peer to send you files."
 
+	// Meshnet set commands group
+	MsgMeshnetSetUsage = "COPY: Sets meshnet configuration option"
+
+	MsgMeshnetSetMachineNicknameUsage = "COPY: set current machine nickname"
```

Comment:
Is it marked with `COPY:` for copywriters to review?

Reply:
yes, I've marked all the new strings with this to find them easier so I don't miss something.